src/resources/js/views/manager.js

```diff
@@ -377,6 +377,11 @@ tribe.events.views.manager = {};
 		var url = target.location.href;
 		var $container = obj.getLastContainer();
 
+		// A pop state from a hash inside of the same URL has been trigger like #content  so no request is required.
+		if ( url && url.split( '#' ).length >= 2 ) {
```

Comment:
2 things here:

1. We can check whether a hash is empty or not by checking `target.location.hash`
2. This takes care of the cases when a url did not have a hash but now has a hash or when a url has a hash and the hash changed to something else. However, it does not address the situation where a url has a hash but now does not., in which case we should not trigger an ajax request either.

Reply:
1. I wasn't aware of that property, so let's use that one instead.
2. I think that's a valid point let me take a look to see if we can put something for that use case.